### ISSUE TITLE
Is PolarMap.js can be set to Antartica ? 

### ISSUE THREAD

**Issue by PBrockmann** [GeoSensorWebLab/polarmap.js#4]
Can the same features be available for Antartica ?

**Comment by liangsteve** [comment]
Yes.  It can be available for Antartica.  Why don't you describe your use case and/or applications?  If there's a strong need, we can see what we can do.

**Comment by PBrockmann** [comment]
Yes of course.
I would like to offer interactive maps for research projects such as http://cdiac.ornl.gov/oceans/GLODAPv2/
Combined with other librairies such as dimensional filtering (dc.js) and interactive plots with bokeh API, you can build very useful applications for the scientific community. See http://ClimateProxiesFinder.ipsl.fr for example, I have recently coded.
I have found PolarMap.js very useful and I was wondering how it would be difficult to add the same features for Antartic region.

**Comment by jenseva** [comment]
I also am hoping to develop an interface that would provide interactive maps for both the arctic and antarctic. I would be integrating real-time json, wms, shape data and would work towards integrating a time slider/animation capability.

**Comment by ghost** [comment]
If anyone's made progress on this, noting it in https://wiki.osm.org/wiki/Antarctica/Creating_a_map could be helpful to others.
